src/Symfony/Component/HttpKernel/Debug/FileLinkFormatter.php

```diff
@@ -75,14 +76,14 @@ private function getFileLinkFormat()
         if ($this->fileLinkFormat) {
             return $this->fileLinkFormat;
         }
-        if ($this->requestStack && $this->baseDir && $this->urlFormat) {
-            $request = $this->requestStack->getMasterRequest();
-            if ($request instanceof Request) {
-                return array(
-                    $request->getSchemeAndHttpHost().$request->getBaseUrl().$this->urlFormat,
-                    $this->baseDir.DIRECTORY_SEPARATOR, '',
-                );
-            }
+
+        if (null === $this->router) {
```

Comment:
If router is null, then we probably need to fallback using the request stack for BC compat.